scripts/us_census/pep/pep_by_asrh/preprocess.py

```diff
@@ -0,0 +1,1053 @@
+    yr_100_df = data_df[data_df[1] == yr_100].iloc[:, 1:].reset_index(drop=True)
+    data_df = data_df[data_df[1] == yr].iloc[:, 1:].reset_index(drop=True)
+    cols = [
+        "Year", "Age", "Total_Population", "Total_Male_Population",
+        "Total_Female_Population", "Male_WhiteAlone_Population",
+        "Female_WhiteAlone_Population",
+        "Male_BlackOrAfricanAmericanAlone_Population",
+        "Female_BlackOrAfricanAmericanAlone_Population",
+        "Male_AmericanIndianAndAlaskaNativeAlone",
+        "Female_AmericanIndianAndAlaskaNativeAlone",
+        "Male_AsianOrPacificIslander", "Female_AsianOrPacificIslander",
+        "Male_HispanicOrLatino", "Female_HispanicOrLatino",
+        "Male_WhiteAloneNotHispanicOrLatino",
+        "Female_WhiteAloneNotHispanicOrLatino",
+        "Male_NotHispanicOrLatino_BlackOrAfricanAmericanAlone",
+        "Female_NotHispanicOrLatino_BlackOrAfricanAmericanAlone",
+        "Male_NotHispanicOrLatino_AmericanIndianAndAlaskaNativeAlone",
+        "Female_NotHispanicOrLatino_AmericanIndianAndAlaskaNativeAlone",
+        "Male_NotHispanicOrLatino_AsianOrPacificIslander",
+        "Female_NotHispanicOrLatino_AsianOrPacificIslander"
+    ]
+    hispanic_cols = _get_mapper_cols_dict("nationals_1980_1999_hispanic")
+    derived_cols = _get_mapper_cols_dict("nationals_1980_1999_derived")
+    data_df.columns = cols
+    # Appending yr_100_df to data_df(which contains age for 1 to 99)
+    if yr_100_df.shape[0] > 0:
+        yr_100_df.insert(0, 0, yr)
+        yr_100_df[1] = 100
+        yr_100_df = yr_100_df.drop(columns=[23])
+        yr_100_df.columns = cols
+        data_df = pd.concat([data_df, yr_100_df]).reset_index(drop=True)
+    # Type casting dataframe values to int.
+    data_df = _convert_to_int(data_df)
+    # Creating Year Column
+    data_df["Year"] = "19" + data_df["Year"].astype('str').str[-2:]
+    # Deriving Columns for HispanicOrLatino Origin
+    for dsv, sv in hispanic_cols.items():
+        data_df[sv[1:]] = -data_df[sv[1:]]
+        data_df[dsv] = data_df.loc[:, sv].sum(axis=1)
+        data_df[sv[1:]] = -data_df[sv[1:]]
+        cols.append(dsv)
+    # Deriving New Columns
+    data_df = _derive_cols(data_df, derived_cols)
+    cols = cols + list(derived_cols.keys())
+    f_cols = [val for val in cols if "Hispanic" in val]
+    data_df = _unpivot_df(data_df, ["Year", "Age"], f_cols)
+    # Creating SV's name using SV, Age Column
+    data_df["SV"] = data_df.apply(lambda row: _create_sv(row.SV, row.Age),
+                                  axis=1)
+    data_df["SV"] = data_df["SV"].str.replace("85OrMore", "85")
+    data_df["Location"] = "country/USA"
+    final_cols = [
+        "Year", "Location", "SV", "Measurement_Method", "Count_Person"
+    ]
+    # Deriving Measurement Method for the SV's
+    data_df = _add_measurement_method(data_df, "SV", "Measurement_Method")
+    return data_df[final_cols]
+
+
+def _process_state_1990_1999(file_path):
```

Comment:
Please group county, state, national functions next to each other and in order of year they are processing data. County looks fine; but national and state are mixed.